tests/E2E Tests/WebAppUiTests/WebAppCallsApiCallsGraphLocally.cs

```diff
@@ -172,6 +176,120 @@ public async Task ChallengeUser_MicrosoftIdFlow_LocalApp_ValidEmailPasswordCreds
                 playwright.Dispose();
             }
         }
+
+        [Fact]
+        [SupportedOSPlatform("windows")]
+        public async Task ChallengeUser_MicrosoftIdFlow_LocalApp_ValidEmailPasswordCreds_TodoAppFunctionsCorrectlyWithCiam()
```

Comment:
What is the difference between this test and the prev one? Does this call the web api and the prev one just "signs in"

Reply:
Both of the tests call the down stream api, the previous test simply adds additional clicks in the app for the todo list feature which is not present in this app.

Reply:
Is it worth having both? Is one a superset of the other?

Reply:
I am not sure this correct. I think the names are confusing. 

TestingWebAppLocally - tests just web app scenario (AcquireTokenByAuthcode / AcquireTokenSilent)
TestingWebAppCallsApiCallGraph - tests web app and web api scenario (so OBO too).

Reply:
I agree, name will be updated. This test involves a web app calling a downstream web api (locally)